app/services/schemas.py

```diff
@@ -54,3 +54,16 @@ class ChatMessage(BaseModel):
     type: str
     text: str
 
+class GivenFiles(BaseModel):
```

Comment:
This is not required anymore as it will be given in the args. of the request.

Reply:
New schema:
```
class NotesGeneratorArgs(BaseModel):
    topic: str = Field(..., description="Topic of the notes")
    details: str = Field(..., description="What should the notes talk about")
    nb_columns: int = Field(..., description="Number of columns for the key concept notes generated")
    orientation: Literal["landscape", "portrait"] = Field(..., description="Orientation of the notes document created")
    file_urls: str = Field(..., description="Comma-separated URLs or paths of the files to be processed")
    file_types: str = Field(..., description="Comma-separated file types corresponding to the provided URLs (e.g., pdf,gdoc,img,youtube_url)")
    langs: str = Field(..., description="Comma-separated languages for each file (e.g., en,fr,es)")


```